acceptor/tcp_acceptor.go

```diff
@@ -117,13 +121,13 @@ func (a *TCPAcceptor) Stop() {
 }
 
 func (a *TCPAcceptor) hasTLSCertificates() bool {
-	return a.certFile != "" && a.keyFile != ""
+	return len(a.certs) > 0
```

Comment:
Shouldn't this be != 2 ? Otherwise it will return success if it only has 1 cert or if it has 10

Reply:
If I recall correctly, I think that the TLS API supports multiple certificates. So in order to make the "new" tcp acceptor API fully functional, I left it as a more generic checking.

The constraint for 2 files is only related to the "legacy" API that loads the files itself. I think we can remove this altogether in the future, making the code much more generic.

What do you think?